### DISCUSSION TITLE
getting error 404 to compute the screenshot by api GET ​/chart​/{pk}​/cache_screenshot​/

### DISCUSSION THREAD

**Discussion by pritypriya25** [apache/superset#15249] [Q&A / Help]
I have loaded my charts in my dashboard. I am entering the slice id but still I am getting the error status 404: Not Found. I am not getting what I am doing wrong. Can someone help me?

**Comment by chenshaojie** [comment]
I'm sorry to bother you. I have the same problem. My English is a little poor. I'm really sorry
I want to get the chart through the API and display it on my website, but my foundation is too poor.

I tried to use the postman tool to get the ID and name of all charts through {/ api / v1 / chart} and tried to find the iframe connection, but the former succeeded and the latter couldn't find it.

So I got access_ token through {/ api / v1 / security / login}。 Store it as material, and I use {/ chart / {PK} / cache _ Screenshot /} but got 404 interface.

I haven't found a way to solve it. Do you have a way to solve this problem now?

Or you can tell me how to get the chart and display it on my website, or get the iframe automatically, instead of manually connecting through < > copy iframe on superset's website.

Thank you very much for taking the time to look at what may be a little stupid.

**Reply by Synarcs** [reply]
@pritypriya25 can you share the logs, I assume it is the issue mostly related to selenium web driver configuration, because  the web driver  may be failing to initiate causing the superset cache to break. On the other hand if the selenium config is fine, then mostly have you enabled thumbnail cache in superset config.
